theme.json

```diff
@@ -415,17 +464,17 @@
 				}
 			},
 			"core/post-content": {
-				"typography": {
-					"lineHeight": "clamp(1.838rem, 1.763rem + 0.249vw, 1.95rem)",
-					"letterSpacing": "-0.01em"
-				},
 				"spacing": {
 					"blockGap": "20px",
 					"padding": {
-						"top": "20px",
-						"bottom": "20px"
+						"top": "1.25rem",
+						"bottom": "1.25rem"
 					}
 				},
+				"typography": {
+					"lineHeight": "clamp(1.838rem, 1.763rem + 0.249vw, 1.95rem)",
+					"letterSpacing": "-0.01em"
+				},
 				"elements": {
 					"link": {
```

Comment:
Links in content don't have hover styles. 
Looks like it's also missing in a few other places - Exit Course block, and outside learning mode in the content, main menu and sidebar links.

Reply:
Good catch! Fixed the ones pertaining to learning mode [here](https://github.com/Automattic/course/pull/68/commits/62b93289217c50c3f55a2357b99489c40a205aad)